src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -431,6 +431,16 @@ export class SbbFormField implements ComponentInterface {
     await this._checkAndUpdateInputEmpty();
   }
 
+  /** Manually clears the input value. */
+  @Method() public async clear(): Promise<void> {
+    if (this._input.tagName !== 'INPUT') {
+      return;
+    }
+    (this._input as { value }).value = '';
+    toggleDatasetEntry(this._element, 'inputEmpty', this._isInputValueEmpty());
+    this._input.focus();
```

Comment:
In this case (if triggered by user clear button) we probably should also dispatch input and change events on the input? I am also interested in the meaning of @kyubisation.